include/support.h

```diff
@@ -82,6 +82,13 @@ inline constexpr T1 ceil_sdivide(const T1 x, const T2 y) noexcept {
 	// https://stackoverflow.com/a/33790603
 }
 
+inline int iround(double x) {
+	assert(std::isfinite(x));
+	assert(x >= (std::numeric_limits<int>::min)());
+	assert(x <= (std::numeric_limits<int>::max)());
+	return static_cast<int>(round(x));
```

Comment:
Can `iround` be fed by user-content? (just thinking about release builds when the `assert`s are gone).  

That said, if it does get user content, I haven't thought about how the release build should behave... maybe an `if` guard, and then E_Exit("... ")?

Reply:
So far no, it's not fed any user content - I use it exclusively for converting `sdl.draw.width * sdl.draw.scalex` (`int` * `double`) back into `int`.

But those asserts illustrate why functions like `round`, `ceil`, `floor` return `double` - we just need to be extra careful whenever doing operations on floating points.

Reply:
Ahh, great! (resolving)